<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RetrieveInstancesWidget</class>
 <widget class="QWidget" name="RetrieveInstancesWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>683</width>
    <height>83</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1,0,0">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QComboBox" name="projectComboBox">
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>0</height>
      </size>
     </property>
     <property name="accessibleName">
      <string>ProjectSelectionComboBox</string>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="filterLineEdit">
     <property name="placeholderText">
      <string>Filter instances</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="allCheckBox">
     <property name="text">
      <string>All instances</string>
     </property>
     <property name="accessibleName">
      <string>AllInstancesCheckBox</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="reloadButton">
     <property name="text">
      <string/>
     </property>
     <property name="accessibleName">
      <string>RefreshInstanceList</string>
     </property>
     <property name="icon">
      <iconset resource="../../icons/orbiticons.qrc">
       <normaloff>:/actions/refresh</normaloff>:/actions/refresh</iconset>
     </property>
     <property name="accessibleName">
      <string>RefreshInstanceList</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../icons/orbiticons.qrc"/>
 </resources>
 <connections/>
</ui>
